eth/transaction_receipt_test.go

```diff
@@ -57,6 +57,7 @@ func TestTransactionReceipts(t *testing.T) {
 
 	require.Equal(t, *eth.MustTopic("0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef"), receipt.Logs[0].Topics[0])
 	require.Equal(t, *eth.MustData256("0x00000001000000000000000000000000000000000000000000008000000000000000000000010000000000000000000000400000000000000000000000000008000000000000000000000008000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000010000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000200000000002000000000000000000000000000000000000000000000000000000000000000008000000000000000000000010000000000000000000000000000000"), receipt.LogsBloom)
+	require.Nil(t, receipt.Type)
```

Comment:
Can't leave a comment in other parts of this file, but should this be checked on the two contract creation tests below?

Reply:
Done in 9127369.  I went further and made `Transaction.transactionType` public and added it to `TransactionReceipt` as well, since consumers will probably find it easier to key off the result of that function than testing if `.Type` is not nil and if so which type.